nerdlets/create-flow/import.js

```diff
@@ -1,24 +1,25 @@
-import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 
-import { Button, HeadingText, Icon, Radio } from 'nr1';
+import { Button, HeadingText, Icon, InlineMessage, Radio } from 'nr1';
 
-import { Select } from '../../src/components';
 import { sanitizeKpis, sanitizeStages } from '../../src/utils';
 
 const fileReader = new FileReader();
 const UPLOADED_FILE_MSG = 'Uploaded file successfully.';
 
-const ImportFlow = ({ accountId, accounts = [], onCreate, onCancel }) => {
+const ImportFlow = ({ accountId, accountName, onCreate, onCancel }) => {
   const [jsonText, setJsonText] = useState('');
   const [selectedAccountId, setSelectedAccountId] = useState();
+  const [selectedAccountName, setSelectedAccountName] = useState();
   const [jsonStatusMsg, setJsonStatusMsg] = useState('');
   const [showDragging, setShowDragging] = useState(false);
   const [structureOnly, setStructureOnly] = useState(false);
 
   useEffect(() => {
     if (!selectedAccountId) setSelectedAccountId(accountId);
-  }, [accountId, selectedAccountId]);
+    if (!selectedAccountName) setSelectedAccountName(accountName);
+  }, [accountId, selectedAccountId, selectedAccountName]);
```

Comment:
See previous comment - remove this block and use `accountName` from the passed props